### ISSUE TITLE
PDF417 doesn't work if RSS14 format is mentioned

### ISSUE THREAD

**Issue by srgade** [phonegap/phonegap-plugin-barcodescanner#489]
### Expected Behaviour
Should support RSS14 and PDF417 without any conflict.
### Actual Behaviour
If RSS14 is mentioned in formats then PDF417 is not supported.
### Reproduce Scenario (including but not limited to)
Mention RSS14 and PDF417 to scan both the codes. Only RSS14 works but not PDF417.
#### Steps to Reproduce
Mention RSS14 and PDF417 to scan both the codes. Only RSS14 works but not PDF417
#### Platform and Version (eg. Android 5.0 or iOS 9.2.1)
Android 5.0
#### (Android) What device vendor (e.g. Samsung, HTC, Sony...)
Samsung Galaxy S5
#### Cordova CLI version and cordova platform version

    cordova --5.1.1v
    cordova platforms - Android and iOS

#### Plugin version

    cordova plugin version | grep phonegap-plugin-barcodescanner
    6.0.6v

**Comment by macdonst** [comment]
@srgade can you provide some sample code that reproduces the problem?

**Comment by srgade** [comment]
In the below sample code, if we have both RSS14 and PDF_417 then only RSS14 tags are scanned successfully. But if we remove RSS14 then PDF_417 formats are scanned.

Sample Code:
`window.cordova.plugins.barcodeScanner.scan((result) => {
      if (result.cancelled) {
        alert("Scan cancelled");
        return;
      }
      const data = result.text.split(/[\n\r]/g);

      if (data.length > 1) {
        alert(`Data Scanned:\n${result.text}`);
        return;
      }
    },
    (err) => alert(`Scanning failed: ${err}`),
      {
        prompt: "Place a barcode inside the scan area", // Android
        formats: "QR_CODE,DATA_MATRIX,UPC_E,UPC_A,EAN_8,EAN_13,CODE_128,CODE_39,CODE_93,CODABAR,ITF,**RSS14**,RSS_EXPANDED,**PDF_417**,AZTEC", // default: all but PDF_417 and RSS_EXPANDED
      });
  },
};
`

**Comment by macdonst** [comment]
@srgade yeah, that's a weird one. I can confirm the problem but I'm pretty sure it's an issue inside the ZXing library.

**Comment by macdonst** [comment]
@srgade not sure just yet. I think the next step is to upgrade the ZXing library the plugin uses.

**Comment by srgade** [comment]
@macdonst  ok. any references for upgrading ?

**Comment by macdonst** [comment]
@srgade I either need to upgrade the pre-compiled jar that @EddyVerbruggen uses or move to a different gradle package.

**Comment by sebafra** [comment]
Same problem. Works perfectly with QR.
But cannot read PDF417
I've tested several cases and nothing happens.

Android 6

`global packages:

    @ionic/cli-utils : 1.5.0
    Cordova CLI      : 7.0.1
    Ionic CLI        : 3.5.0

local packages:

    @ionic/app-scripts              : 1.3.7
    @ionic/cli-plugin-cordova       : 1.4.1
    @ionic/cli-plugin-ionic-angular : 1.3.2
    Cordova Platforms               : android 6.2.3 ios 4.4.0
    Ionic Framework                 : ionic-angular 3.3.0

System:

    Node       : v6.9.2
    OS         : OS X El Capitan
    Xcode      : Xcode 8.0 Build version 8A218a
    ios-deploy : 1.9.0
    ios-sim    : 5.0.13
    npm        : 4.0.5`
